src/SFA.DAS.ApplyService.Data/Queries/AppealsQueries.cs

```diff
@@ -73,13 +53,32 @@ public async Task<Appeal> GetAppeal(Guid applicationId, Guid oversightReviewId)
 
                             return result;
                         }, new
-                    {
-                        applicationId,
-                        oversightReviewId
-                    });
+                        {
+                            applicationId
+                        });
 
                 return result;
             }
         }
+
+        public async Task<AppealFiles> GetStagedAppealFiles(Guid applicationId)
+        {
+            using (var connection = GetConnection())
+            {
+                var files = (await connection.QueryAsync<AppealFile>(
+                    @"SELECT Id, Filename FROM [AppealUpload] where ApplicationId = @applicationId and AppealId IS NULL ORDER BY CreatedOn ASC",
```

Comment:
change this to left outer join on appeal for checking no appeal in process

Reply:
That's for APR-2640, not this ticket. Have noted it 👍